paasta_tools/tron_tools.py

```diff
@@ -327,15 +327,31 @@ def format_tron_job_dict(job_config, cluster_fqdn_format, default_paasta_cluster
     # Only pass non-None values, so Tron will use defaults for others
     return {key: val for key, val in result.items() if val is not None}
 
+def _update_config(config, tron_conf_path):
+    tron_config_file = service_configuration_lib._read_yaml_file(tron_conf_path)
+    if tron_config_file is not None:
+        for key, value in tron_config_file.items():
+            if isinstance(value, list):
+                config[key] += value
+            else:
+                config[key] = value
 
 def load_tron_service_config(service, tron_cluster, soa_dir=DEFAULT_SOA_DIR):
     """Load all configured jobs for a service, and any additional config values."""
-    tron_conf_file = os.path.join(
+    config = defaultdict(list)
+    
+    tron_conf_path1 = os.path.join(
+        os.path.abspath(soa_dir), service, 'tron-' + tron_cluster + '.yaml'
+    )
+    _update_config(config, tron_conf_path1)
+
+    tron_conf_path2 = os.path.join(
         os.path.abspath(soa_dir), 'tron', tron_cluster, service + '.yaml',
     )
-    config = service_configuration_lib._read_yaml_file(tron_conf_file)
-    if not config:
-        raise NoConfigurationForServiceError('No configuration found for Tron at %s' % tron_conf_file)
+    _update_config(config, tron_conf_path2)
```

Comment:
Does this mean we might end up with configuration for the same namespace in two different places? I don't think that we should let that happen. We should just check the two files in order, and return the first config that exists.

Reply:
If so, it will require we migrate the entire namespace against each cluster once. I presume it's the common case, but it won't hurt to keep some flexibility. Unless you have a strong opinion on that?

Reply:
I think it will be confusing to have 2 different locations where you can make changes to your tronfig for the same namespace.

For big namespaces like MASTER, the service name (yelp-main, etc) won't match the old namespace (MASTER) so you can still move things out of MASTER incrementally.